public/styles/components/campaign-info/analytics/_analytics.scss

```diff
@@ -0,0 +1,72 @@
+.campaign-info {
+  width: 100%;
+
+}
+
+.analytics-chart__full-width {
+
```

Comment:
These should probably have a single pseudo class %analytics-chart

Then the different sizes can be based on that class

```
.analytics-chart--full-width {
 @extend %analytics-chart;
 ...
}
```